lib/reporters/console.js

```diff
@@ -13,6 +14,12 @@ const internals = {
     width: 50
 };
 
+internals.stringify = (obj, serializer, indent, decycler) => {
+
+    return StableStringify(
```

Comment:
Can't we just do `StableStringify(obj, { space: indent, replacer: StringifySafe.getSerialize(serializer) })` ?